### ISSUE TITLE
Themes for CSAT Helpdesk & Newsletter surveys aren't good on Mobile devices

### ISSUE THREAD

**Issue by pavithra-opg** [OnePointGlobal/OnePoint-Global-Account-Toolkit#353]
Hi Guys,

Theme used in CSAT Helpdesk & Newsletter survey is good in Web browser but looks awful in device.
Survey links are below for your reference:
CSAT_Helpdesk:
http://1pt.mobi/f/E32

CSAT_Newsletter:
http://1pt.mobi/f/E33

Can this be corrected?

![image1](https://user-images.githubusercontent.com/28919270/31610085-5a032a48-b294-11e7-8faf-c6f805a00679.PNG)

**Comment by dougmorris** [comment]
@pavithra-opg is this still a requirement?

**Comment by pavithra-opg** [comment]
@dougmorris : yes, it is a Live survey which is used to get feedback from our clients. Not an urgent requirement but it will give a good user experience if fixed.
